installIODTB.bat

```diff
@@ -0,0 +1,24 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+mvn clean package -pl client -am -DskipTests
+mvn clean package -pl server -am -DskipTests
+mvn clean package -pl zeppelin-interpreter -am -DskipTests
```

Comment:
> Could this file be deleted? It seems to be shortcut of commands : )

I agree. Hi @dynamicDr ,  what's it for? shall we delete it or add them into `README`?

Reply:
> > Could this file be deleted? It seems to be shortcut of commands : )
> 
> I agree. Hi @dynamicDr , what's it for? shall we delete it or add them into `README`?

Maybe we could delete it first so that this PR could be merged 😄

Reply:
Sure, I have removed it.